lms/djangoapps/discussion_api/api.py

```diff
@@ -263,20 +279,38 @@ def get_thread_list(request, course_key, page, page_size, topic_id_list=None, te
     if exclusive_param_count > 1:  # pragma: no cover
         raise ValueError("More than one mutually exclusive param passed to get_thread_list")
 
+    # These should already be validated by the django forms.
```

Comment:
Agreed. I'm going add something more descriptive about this in the docstring instead.